src/v/cluster/topics_frontend.cc

```diff
@@ -487,6 +489,32 @@ ss::future<topic_result> topics_frontend::do_create_topic(
               "valid "
               "topic manifest");
         }
+        auto validation_map = co_await maybe_validate_recovery_topic(
+          assignable_config, bucket, _cloud_storage_api.local(), _as.local());
+        if (std::ranges::any_of(
```

Comment:
Should we consider the timeout here too?

Reply:
you mean the timeout for the create_topics operations? 
yes we should (https://github.com/redpanda-data/core-internal/issues/1221) 

in the ducktape local environment, and from some testing also on a cluster with s3, the current defaults seems to be ok.

setting the cluster default validation to no_check would work as an escape hatch in case this operation becomes the bottleneck.

maybe i can return an optional<map<partition_id, validation_res>> to remove the runtime cost in case of nocheck